util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -79,33 +78,22 @@ std::unique_ptr<RequestBuffer> RequestBuffer::New() {
   return std::unique_ptr<RequestBuffer>(new RequestBufferImpl);
 }
 
-static bool ReadAll(int fd, void *in_data, size_t data_len) {
-  uint8_t *data = reinterpret_cast<uint8_t *>(in_data);
-  size_t done = 0;
+static bool ReadAll(std::istream *stream, void *in_data, size_t data_len) {
+  size_t read = stream->read(static_cast<char *>(in_data), data_len).gcount();
```

Comment:
Is there a reason we switched from uint8_t to char?

Reply:
Stream read/write work on `char`s while read/write expect `uint8_t` or `void *`.